package.json

```diff
@@ -6,29 +6,28 @@
-    "compile:wallet": "npx etherlime compile --workingDirectory contracts/wallet --runs=999",
-    "compile:test": "npx etherlime compile --workingDirectory contracts-test --runs=999",
+    "compile:modules": "npx truffle compile --config truffle-config-modules.js && rm build/contracts/ApprovedTransfer.json && npx truffle compile --contracts_directory=./contracts/modules/ApprovedTransfer*.sol && rm build/contracts/RelayerManager.json && npx truffle compile --contracts_directory=./contracts/modules/RelayerManager*.sol && rm build/contracts/TransferManager.json && npx truffle compile --contracts_directory=./contracts/modules/TransferManager*.sol && rm build/contracts/TokenExchanger.json && npx truffle compile --contracts_directory=./contracts/modules/TokenExchanger*.sol",
+    "compile:wallet": "npx truffle compile --config truffle-config-wallet.js",
+    "compile:test": "npx truffle compile --config truffle-config-contracts-test.js",
     "compile": "npm run compile:infrastructure && npm run compile:modules && npm run compile:wallet",
-    "cc": "rm -rf build && rm -rf build-legacy && npm run compile:lib && npm run compile && npm run compile:legacy && npm run compile:test",
-    "ganache": "npx etherlime ganache --gasLimit=20700000 -e 10000",
+    "cc": "rm -rf build && rm -rf build-legacy && npm run compile:lib && npm run compile && npm run compile:legacy && npm run compile:test && npm run provision:lib:artefacts",
+    "ganache": "npx ganache-cli --chainId 1895 --networkId 1597649375983 --gasLimit=20700000 -e 10000 --acctKeys=\"./ganache-accounts.json\" --account=\"0x7ab741b57e8d94dd7e1a29055646bafde7010f38a900f55bbd7647880faa6ee8, 100000000000000000000000000000000000000\" --account=\"0x2030b463177db2da82908ef90fa55ddfcef56e8183caf60db464bc398e736e6f, 100000000000000000000000000000000000000\" --account=\"0x62ecd49c4ccb41a70ad46532aed63cf815de15864bc415c87d507afd6a5e8da2, 100000000000000000000000000000000000000\" --account=\"0xf473040b1a83739a9c7cc1f5719fab0f5bf178f83314d98557c58aae1910e03a, 100000000000000000000000000000000000000\" --account=\"0x823d590ed2cb5e8493bb0efc834771c1cde36f9fc49b9fe3620ebd0754ad6ea2, 100000000000000000000000000000000000000\" --account=\"0xd6d710943471e4c37ceb787857e7a2b41ca57f9cb4307ee9a9b21436a8e709c3, 100000000000000000000000000000000000000\" --account=\"0x187bb12e927c1652377405f81d93ce948a593f7d66cfba383ee761858b05921a, 100000000000000000000000000000000000000\" --account=\"0xf41486fdb04505e7966c8720a353ed92ce0d6830f8a5e915fbde735106a06d25, 100000000000000000000000000000000000000\" --account=\"0x6ca40ba4cca775643398385022264c0c414da1abd21d08d9e7136796a520a543, 100000000000000000000000000000000000000\" --account=\"0xfac0bc9325ad342033afe956e83f0bf8f1e863c1c3e956bc75d66961fe4cd186, 100000000000000000000000000000000000000\"",
```

Comment:
Wouldn't it be easier to run ganache-cli in deterministic mode (flag -d) and use these accounts instead of importing (and exporting) etherlime accounts?

Reply:
good point. updated